File: front/Database.Migration.System/src/app/api/models/current-migration-settings-dto.ts
/* tslint:disable */
/* eslint-disable */
import { DatabaseType } from './database-type';
export interface CurrentMigrationSettingsDto {
  destinationDatabaseDataInfo?: null | string;
  destinationDatabaseType?: DatabaseType;
  sourceDatabaseDataInfo?: null | string;
  sourceDatabaseType?: DatabaseType;
}

File: DatabaseMigrationSystem.Common/Dto/CurrentMigrationSettingsDto.cs
using DatabaseMigrationSystem.Common.Enums;

namespace DatabaseMigrationSystem.Common.Dto;

public class CurrentMigrationSettingsDto
{
    /// <summary>
    /// 
    /// </summary>
    public string SourceDatabaseDataInfo  { get; set; }
    
    /// <summary>
    /// Тип базы данных
    /// </summary>
    public DatabaseType SourceDatabaseType { get; set; }
    
    /// <summary>
    /// 
    /// </summary>
    public string DestinationDatabaseDataInfo  { get; set; }
    
    /// <summary>
    /// Тип базы данных
    /// </summary>
    public DatabaseType DestinationDatabaseType { get; set; }
}